charts/digital-product-pass/values-beta.yaml

```diff
@@ -78,12 +78,11 @@ backend:
         hosts:
           - materialpass.beta.demo.catena-x.net
 
-  avp:
-    helm:
-      clientId: <path:material-pass/data/beta/backend#clientId>
-      clientSecret: <path:material-pass/data/beta/backend#clientSecret>
-      xApiKey: <path:material-pass/data/beta/edc/oauth#api.key>
-      participantId:  <path:material-pass/data/beta/edc/participant#bpnNumber>
+  edc:
+    clientId: <path:material-pass/data/dev/backend#clientId>
+    clientSecret: <path:material-pass/data/dev/backend#clientSecret>
+    xApiKey: <path:material-pass/data/dev/edc/oauth#api.key>
+    participantId:  <path:material-pass/data/dev/edc/participant#bpnNumber
```

Comment:
missing closing parenthesis ">" for the participantId.

Reply:
values beta has dev secret references

Reply:
The typo is fixed, closing this conversation